charts/spire/charts/spire-server/values.yaml

```diff
@@ -562,6 +562,27 @@ tornjak:
     ## @param tornjak.service.annotations [object] Annotations for the service
     annotations: {}
 
+  ## @param tornjak.ingress.enabled Flag to enable ingress for Tornjak backend service
+  ## @param tornjak.ingress.className Ingress class name for Tornjak backend service
+  ## @param tornjak.ingress.annotations [object] Annotations for Tornjak backend service
+  ## @param tornjak.ingress.servicePort Service port value for Tornjak backend service
+  ## @param tornjak.ingress.hosts [array] Host paths for ingress Tornjak backend service
+  ingress:
+    enabled: false
+    className: ""
+    annotations: {}
+    servicePort: 10000
```

Comment:
```suggestion
```

This duplicates the port number across the service config and the ingress, which requires consumers to keep it aligned. It is better to use the port name in the ingress as defined in the service, see my other comment.